data/json/monsters/cyborgs.json

```diff
@@ -85,5 +85,131 @@
+    "type": "MONSTER",
+    "name": "Exodii quadruped",
+    "description": "This enormous quadrupedal robot seems to be cobbled together from parts, most of them unfamiliar to you.  It moves with a heavy, oddly graceful gait, its footsteps leaving shallow craters behind.  It bristles with an arsenal of weaponry, but doesn't seem in a particular rush to target you.",
+    "default_faction": "exodii",
+    "volume": "413 L",
+    "weight": "517 kg",
+    "species": [ "CYBORG" ],
+    "bodytype": "dog",
+    "material": [ "steel" ],
+    "speed": 110,
+    "symbol": "M",
+    "aggression": -5,
+    "morale": 70,
+    "hp": 240,
+    "color": "light_gray",
+    "melee_skill": 10,
+    "melee_dice": 3,
+    "melee_dice_sides": 8,
+    "melee_cut": 12,
+    "armor_bash": 42,
+    "armor_cut": 48,
+    "vision_day": 50,
+    "vision_night": 50,
+    "revert_to_itype": "e_scrap",
+    "starting_ammo": { "123ln": 3000 },
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "special_attacks": [
+      [ "FLAMETHROWER", 10 ],
+      {
+        "type": "gun",
+        "cooldown": 1,
+        "move_cost": 150,
+        "gun_type": "pamd68",
+        "ammo_type": "123ln",
+        "fake_skills": [ [ "gun", 5 ], [ "rifle", 8 ] ],
+        "fake_dex": 12,
+        "ranges": [ [ 0, 41, "DEFAULT" ] ],
+        "require_targeting_npc": true,
+        "require_targeting_monster": true,
+        "laser_lock": false,
+        "targeting_cost": 400,
+        "targeting_timeout_extend": -10,
+        "no_ammo_sound": "a chk!"
+      }
+    ],
+    "death_drops": { "subtype": "collection", "groups": [ [ "robots", 80 ] ] },
+    "death_function": [ "BROKEN" ],
+    "flags": [ "SEES", "HEARS", "GOODHEARING", "NOHEAD", "NO_BREATHE", "PATH_AVOID_DANGER_2", "PRIORITIZE_TARGETS", "HIT_AND_RUN" ]
+  },
+  {
+    "id": "mon_zomborg",
+    "type": "MONSTER",
+    "name": "zomborg",
+    "description": "A mix of dead human and even deader technology, this twisted mess of steel and flesh moves like a puppet in the hands of an angry toddler.  Its robotic components seem to have shut down, and new bands of flesh have wrapped around them, tugging and pulling them in awkward directions.  Bits of metallic skeleton and armor plating jut from its decaying flesh.",
+    "default_faction": "zombie",
+    "volume": "120 L",
+    "weight": "200 kg",
+    "species": [ "CYBORG", "ZOMBIE" ],
+    "bodytype": "human",
+    "material": [ "steel", "flesh" ],
+    "speed": 60,
+    "symbol": "Z",
+    "morale": 100,
+    "aggression": 100,
+    "hp": 125,
+    "color": "light_gray",
+    "melee_skill": 3,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_cut": 12,
+    "armor_bash": 12,
+    "armor_cut": 28,
+    "vision_night": 3,
+    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "GRAB", 7 ], [ "scratch", 20 ] ],
+    "death_drops": { "subtype": "collection", "groups": [ [ "robots", 80 ] ] },
+    "death_function": [ "FIREBALL" ],
```

Comment:
Does that mean the zomborg explodes in flame upon death? And if yes, why?

Reply:
Sometimes (1/10 times). Some of them still have functional, unstable plutonium cells that they can't use.